.stylelintrc.json

```diff
@@ -1,97 +1,37 @@
 {
   "extends": "stylelint-config-standard-scss",
   "rules": {
-	"alpha-value-notation": null,
+	"declaration-colon-newline-after": null,
+	"selector-list-comma-newline-after": null,
```

Comment:
Rules that reformat the code a lot and mess things up.

`declaration-colon-newline-after` just messes up some CSS. :frowning_face: 

`selector-list-comma-newline-after`:

```
one, two, three
```
becomes
```
one,
 two,
 three
```
(And depends on re-indention)